internal/primitive/transform/action/strings/split_with_intervals.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 Linkall Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package strings
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/linkall-labs/vanus/internal/primitive/transform/action"
+	"github.com/linkall-labs/vanus/internal/primitive/transform/arg"
+	"github.com/linkall-labs/vanus/internal/primitive/transform/common"
+	"github.com/linkall-labs/vanus/internal/primitive/transform/context"
+)
+
+type splitWithIntervalsAction struct {
+	action.CommonAction
+}
+
+// NewSplitWithIntervalsAction["sourceJSONPath", "startPosition", "splitInterval", "targetJsonPath"].
+func NewSplitWithIntervalsAction() action.Action {
+	return &splitWithIntervalsAction{
+		CommonAction: action.CommonAction{
+			ActionName: "SPLIT_WITH_INTERVALS",
+			FixedArgs:  []arg.TypeList{arg.EventList, arg.All, arg.All, arg.EventList},
```

Comment:
```suggestion
			FixedArgs:  []arg.TypeList{arg.EventList, arg.All, arg.All, []arg.Type{arg.EventData}},
```
the target value is an array, the [event attribute](https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md#type-system)  doesn't support the array, so the target path must be  event data